### ISSUE TITLE
Peacock does not maintain input file's comment location

### ISSUE THREAD

**Issue by hugary1995** [idaholab/moose#11732]
## Rationale
This is a minor minor complaint about Peacock messing up comments in input file.

## Description
Complaint: Often times I make comments in my input file using '#'. Not very often though, I open my input file in Peacock to make a couple of changes and then save the input file -- every time I do that, my original comments are messed up. Currently if I have comments floating around, after saving the input using Peacock, they are all squashed onto the very top of the input file.

## Impact
Fixing this will make my (or some other users') workflow merrier. 😄

**Comment by permcody** [comment]
That's correct it doesn't, but that's because of the very loose format design. As you are aware, you can put comments just about anywhere within blocks, around blocks, before, after parameters, etc. We are working on a system to try to keep comments closer to where we think they belong and we have a little ways to go. I believe the design we are working towards is that comments just inside the block belong to the block itself. Every other comment will bind to the parameter coming immediately after it. 

We are developing a tool that will reformat input files. Much like clang-format reformats your source files. Once that is out, we'll have more consistent looking files throughout all of the framework and the modules so you might not get too picky about where your comments are (e.g. right at a specific column after a parameter for instance).

**Comment by hugary1995** [comment]
We dropped peacock support.
